### PR TITLE
[Dashboard] Updated module help text

### PR DESCRIPTION
**Pull request by knoll-alex**
## Brief summary of changes

New copy for LORIS dashboard (homepage) help text

#### Testing instructions (if applicable)

1. Edit copy if necessary
2. Check out screenshot of rendering below

#### Links to related tickets (GitHub, Redmine, ...)

*https://github.com/aces/Loris/issues/5576

Screenshot 2019-11-11 16 50 37

### CODE REVIEW

File: modules/dashboard/help/dashboard.md [hunk 1]
```diff
@@ -2,39 +2,32 @@
...
-### My Tasks
+#### My Tasks
 
 User-specific list of outstanding tasks such as data entry conflicts, incomplete forms, or new and pending scans awaiting review. Click on any task panel to jump to the appropriate module.
 
-### Document Repository Notifications
+#### Document Repository Notifications
 
-View recently uploaded documents for your study or site user's study or site.  Click on a document name to download the file directly from the Dashboard. Click on "Document Repository" to view the complete list of available documents.
+Recently-uploaded documents for your study or site user's study or site. Click on a document name to download the file directly from the Dashboard. Click **Document Repository** to view the complete list of available documents.
```
Comment (johnsaigle) [inline modules/dashboard/help/dashboard.md]: > for your study or site user's study or site

This sentence is a little hard to parse imo. Otherwise looks good!

Comment (knoll-alex) [reply modules/dashboard/help/dashboard.md]: thanks, done!

File: modules/dashboard/help/dashboard.md [hunk 2]
```diff
@@ -2,39 +2,32 @@
...
-
-To log out from LORIS, click on the user's name and select "Log Out" from the dropdown menu, or select "User Preferences" to update user information, profile settings, and to change your password.
-
-Click on any of the six main menus to access LORIS modules.
+Need to update your account information, such as your password or other profile settings? Click on your name in the top right corner and click **User Preferences**. 
 
 ## Dashboard Features
 
-The dashboard is composed of panels showing at-a-glance status of the LORIS study. It has panels for each of: Recruitment, Study Progression, My Tasks, Document Repository Notifications, and Behavioral Feedback Notifications. Each panel can be hidden by clicking the arrow located in the top right corner.
+This is where you’ll find an at-a-glance status of your LORIS study. Note that all data shown in these sections are from ineligible, excluded, and consent-withdrawn candidates. You can temporarily hide any of the sections from view by clicking the arrow located in the top right corner. We’ve broken down the different sections below:
```
Comment (driusan) [inline modules/dashboard/help/dashboard.md]: > Note that all data shown in these sections are from ineligible, excluded, and consent-withdrawn candidates.

That doesn't sound right..

Comment (johnsaigle) [reply modules/dashboard/help/dashboard.md]: What would be better?

Comment (driusan) [reply modules/dashboard/help/dashboard.md]: Having the dashboard show data from valid candidates instead of ineligible/excluded/consent-withdrawn ones?

Comment (johnsaigle) [reply modules/dashboard/help/dashboard.md]: Is that what it does or is that what it should do? Genuinely asking.

We should have the documentation reflect the current behaviour.

Comment (driusan) [reply modules/dashboard/help/dashboard.md]: Hopefully it's what it does.. if the dashboard is only showing the recruitment graphs and numbers for invalid candidates, it's a pretty serious bug with the dashboard.

Comment (johnsaigle) [reply modules/dashboard/help/dashboard.md]: @knoll-alex The original text says

> Note that charts include data from ineligible, excluded, and consent withdrawn candidates

Which is quite different from saying that ALL the data are from these candidates. 

Could you explain the change? @christinerogers maybe you can weigh in here if this was a collaborative edit.

Comment (knoll-alex) [reply modules/dashboard/help/dashboard.md]: totally my bad - must have misunderstood the original wording but this makes much more sense. Just edited!
